src/commcare_cloud/ansible/roles/nginx/vars/blobdb.yml

```diff
@@ -0,0 +1,41 @@
+# based off https://blog.sentry.io/2017/03/01/dodging-s3-downtime-with-nginx-and-haproxy.html
+# https://gist.github.com/mikhailov/9639593
+# https://stackoverflow.com/questions/44639182/nginx-proxy-amazon-s3-resources
+---
+nginx_sites:
+- support_letsencrypt: false
```

Comment:
would it make sense to add riak to this as well? (means we'd have two riak proxies for icds)

Reply:
Do you mean adding Riak to this proxy node or adding the riak config to this site?

Reply:
Either one.

Reply:
In the interest of fewer changes I'm going to leave riak proxy were it is for now